README.md

```diff
@@ -1,4 +1,4 @@
-<p align="center"><a href="https://github.com/proximax-storage" target="_blank"><img width="150" src="https://github.com/proximax-storage/java-xpx-chain-sdk/raw/master/docs/images/logo.jpg" alt="ProximaX Sirius Blockchain Java SDK"></a></p>
+<p align="center"><a href="https://github.com/proximax-storage" target="_blank"><img width="300" srchttps://user-images.githubusercontent.com/29048783/72933958-c8836400-3d30-11ea-99d5-678e20f43e19.png" alt="ProximaX Sirius Blockchain Java SDK"></a></p>
```

Comment:
the change breaks the content. Image source is not specified correctly. please fix `src` in the `img` tag
`srchttps://user-images.githubusercontent.com/29048783/72933958-c8836400-3d30-11ea-99d5-678e20f43e19.png"`
needs to change to 
`src="https://user-images.githubusercontent.com/29048783/72933958-c8836400-3d30-11ea-99d5-678e20f43e19.png"`